### PR TITLE
PP-504/remove-worker-configuration

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- Expose the getPingResponse

## Why

- To be able to use it in the SDK

## Refs

- [Jira 504](https://rsklabs.atlassian.net/browse/PP-504)
- https://github.com/rsksmart/rif-relay-contracts/pull/91
- https://github.com/rsksmart/rif-relay-common/pull/43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     ]
   },
   "dependencies": {
-    "@rsksmart/rif-relay-common": "1.0.0-alpha.1",
-    "@rsksmart/rif-relay-contracts": "1.0.0-alpha.0",
+    "@rsksmart/rif-relay-common": "github:rsksmart/rif-relay-common#PP-504/remove-worker-configuration",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-504/remove-worker-configuration",
```
Comment (antomor) [inline package.json:31]: Once we merge the related PRs, we need to update it

Comment (jurajpiar) [inline package.json:31]: I'm guessing that those two libs should be merged before this one and these two lines changed back increasing the tag, right?

Comment (jurajpiar) [reply package.json:31]: oops spamming sorry

File: src/api/ExchangeApi.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import BigNumber from 'bignumber.js';
 
+// All FromCurrency tokens should be in uppercases as we uppercase it before doing the request
 export type FromCurrency = 'TRIF' | 'RIF' | 'RDOC' | 'RBTC' | 'TKN';
```
[the remark this answers is not in the document]
Comment (franciscotobar) [reply src/api/ExchangeApi.ts]: updated

File: package.json (hunk 1 above)
Comment (franciscotobar) [reply package.json]: updated

**Review by antomor — APPROVED** [review]
LG

**Pull request merged**